README.md

```diff
@@ -26,18 +26,18 @@ cd open-source-edu-bot
 python3 -m venv venv
 ```
 
-on Windows
+On Windows
 
 ```PowerShell
 py -3 -m venv venv
 ```
 
 **Activate the environment**
 
-Copy the environments file and adapt it:
+Rename the environments file and adapt it:
 
 ```bash
-cp .sample.env .env
+rename .sample.env .env
```

Comment:
Not `rename` but `copy`

Reply:
@vj-codes this should not change! you could instead add 2 examples, one for *unix users (using `cp` cmd) and another one for windows! how does it sound ?

Reply:
> @vj-codes this should not change! you could instead add 2 examples, one for *unix users (using `cp` cmd) and another one for windows! how does it sound ?

Great I'll change it! Sorry got confused:)

Reply:
yes

Reply:
same remark as [here](https://github.com/fbdevelopercircles/open-source-edu-bot/pull/50#discussion_r460918707)!